File: Programmers/Lv3/Lv3_저울.py
def solution(weight):
    answer = 1
    weight.sort()

    for i in weight:
        if answer < i:      # ?꾩쟻??媛믩낫???щ㈃ 留뚮뱾???녿뒗 臾닿쾶??
            break
        answer += i
    return answer

File: Programmers/Lv3/Lv3_저울.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> weight) {
	int answer = 1;
	sort(weight.begin(), weight.end());
	
	for (int i = 0; i < weight.size(); i++) {
		if (answer < weight[i])				// 누적합보다 작으면 탈출
			break;
		answer += weight[i];			// 누적합
	}
	 
	return answer;
}
